src/components/Navbar/Navbar.jsx

```diff
@@ -63,6 +67,20 @@ function Navbar() {
           <small>Home</small>
         </div>
 
+        <div className="flex-col-center">
+          {token ? (
```

Comment:
Instead of having this condition check here, you can move it to `to` attribute of `NavLink` as rest of the things are same.

Reply:
Yess that make sense